docs/ARCHITECTURE.rst

```diff
@@ -937,3 +937,50 @@ Algorithms
 The algorithms for each listing type can be found in the open-source
 `scality/Arsenal <https://github.com/scality/Arsenal>`__ repository, in
 `lib/algos/list <https://github.com/scality/Arsenal/tree/master/lib/algos/list>`__.
+
+Encryption
+===========
+
+About Server Side Encryption CloudServer has its own bucket level encryption.
+
+Differentiations
+------------------
+
+With the real AWS S3, to enable encryption, you have to specify on the object
+level a ``x-amz-server-side-encryption: AES256`` header when you upload an object
+
```

Comment:
Please note that your description above is not correct.  The user does not send the header -- we do.